docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md

```diff
@@ -4,94 +4,160 @@ permalink: ru/
+
+Пример настройки параметров модуля `kube-dns`:
+```yaml
+data:
+  kubeDns: |
+    stubZones:
+    - upstreamNameservers:
+      - 192.168.121.55
+      - 10.2.7.80
+      zone: directory.company.my
+    upstreamNameservers:
+    - 10.2.100.55
+    - 10.2.200.55
+```
+
+### Включение и отключение модуля
+
+> Некоторые модули могут быть включены по умолчанию в зависимости от используемого [набора модулей](#наборы-модулей).
+
+Для включения или отключения модуля необходимо добавить в ConfigMap `deckhouse` параметр `<moduleName>Enabled`, который может принимать одно из двух значений: `"true"` или `"false"` (кавычки обязательны), где `<moduleName>` — название модуля в camelCase.
+
+Пример включения модуля `user-authn`:
 ```yaml
 data:
   userAuthnEnabled: "true"
 ```
 
+## Наборы модулей
+
+Deckhouse работает только с включёнными модулями.
+
+В зависимости от используемого [набора модулей](./modules/020-deckhouse/configuration.html#parameters-bundle) модули могут быть включены или выключены по умолчанию.
+
+{%- assign bundles = site.data.bundles | sort %}
+<table>
+<thead>
+<tr><th>Название набора модулей</th><th>Список включенных по умолчанию модулей</th></tr></thead>
+<tbody>
+{% for bundle in bundles %}
+<tr>
+<td><strong>{{ bundle[0] |  replace_first: "values-", "" | capitalize }}</strong></td>
+<td>{% assign modules = bundle[1] | sort %}
+<ul style="columns: 3">
+{%- for module in modules %}
+{%- assign moduleName = module[0] | regex_replace: "Enabled$", '' | camel_to_snake_case | replace: "_", '-' %}
+{%- assign isExcluded = site.data.exclude.module_names | where: "name", moduleName %}
+{%- if isExcluded.size > 0 %}{% continue %}{% endif %} 
+{%- if module[1] != true %}{% continue %}{% endif %}
+<li>
+{{ module[0] | regex_replace: "Enabled$", '' | camel_to_snake_case | replace: "_", '-' }}</li>
+{%- endfor %}
+</ul>
+</td>
+</tr>
+{%- endfor %}
+</tbody>
+</table>
+
 ## Выделение узлов под определенный вид нагрузки
 
-Для всех модулей принята единая стратегия:
+Если в параметрах модуля не указаны явные значения `nodeSelector/tolerations`, то для всех модулей используется следующая стратегия:
 
-1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с лейблами из определенного списка или лейблами определенного формата (подробнее — ниже, под спойлером), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
-1. Если параметр `tolerations` модуля не указан, то Pod'ам модуля автоматически устанавливаются все возможные toleration'ы (подробнее — ниже, под спойлером);
+1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```

Comment:
```suggestion
1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```
I'm not sure about this edit, but it seems to me that the preposition is not needed here.